libraries/ui/src/utils/auth.test.tsx

```diff
@@ -89,36 +94,6 @@ describe('auth', () => {
       expect(state.auth).toEqual(auth);
     });
 
-    test('should keep new token when replacing soon-to-expire token', () => {
-      setupMockOidcClient(); // Set up OIDC client for potential refresh attempts
-
-      // Set initial auth that expires in 30 seconds
-      const initialAuth = createAuth({
-        expiresAt: Date.now() + 30_000,
-        token: 'initial-token',
-      });
-      useAuthStore.getState().setAuth(initialAuth);
-
-      // Advance 20 seconds (10 seconds before initial token expires)
-      vi.advanceTimersByTime(20_000);
-
-      // Set new auth with fresh expiry
-      const newAuth = createAuth({
-        expiresAt: Date.now() + 3600_000, // 1 hour
-        token: 'new-token',
-      });
-      useAuthStore.getState().setAuth(newAuth);
-
-      // Advance 10 seconds (past when initial token would have expired)
-      vi.advanceTimersByTime(10_000);
-
-      // Should still have the new token (old expiry timer should have been cancelled)
```

Comment:
The old expiry timer is no longer a concern with the new logic, as there is always just one setInterval running using whatever the latest token is